### PR TITLE
Fix event clean method (don't compare time if it is None) + Changes in validation.

### PR DESCRIPTION
**Pull request by cyberstar**
Change validation logic a little bit - more explicit checks + comments.

Should fix issue #54.

### CODE REVIEW

File: aldryn_events/models.py [hunk 1]
```diff
@@ -194,13 +194,26 @@ def clean(self):
...
+                # check that time is provided
+                if not (self.end_time and self.start_time):
+                    raise ValidationError(
+                        _('If you specify same dates for start and end of event'
+                          ' please provide also start and end time. OR you can'
+                          ' make end date to be different'))
+
+                # check time validity
+                if (self.end_time < self.start_time or
+                            self.start_time == self.end_time):
```
Comment (intellectronica) [inline aldryn_events/models.py]: weird indentation. the second line of the conditional should align with the first.

File: aldryn_events/models.py [hunk 2]
```diff
@@ -194,13 +194,26 @@ def clean(self):
...
+        if self.start_date and self.end_date:
+            if self.end_date < self.start_date:
+                raise ValidationError(_('Start date should be before end date.'))
+
+            # dates are equal, check time
+            if self.start_date == self.end_date:
+                # check that time is provided
+                if not (self.end_time and self.start_time):
+                    raise ValidationError(
+                        _('If you specify same dates for start and end of event'
```
Comment (intellectronica) [inline aldryn_events/models.py]: How about:
```
When specifying identical start and end dates, please also provide the start and end time.
```

**Comment by intellectronica** [comment]
:+1:

File: aldryn_events/models.py (hunk 1 above)
Comment (mkoistinen) [inline aldryn_events/models.py]: I'm ok with this, as long as it passes PEP8, I'm ok =)

File: aldryn_events/models.py (hunk 2 above)
Comment (mkoistinen) [inline aldryn_events/models.py]: +1 @intellectronica

**Comment by mkoistinen** [comment]
LGTM

**Pull request merged**